src/Hydrators/IssueHydrator.php

```diff
@@ -48,16 +48,50 @@ public static function getIssue($db, $issue_id)
             return null;
         }
 
-        $commentsQuery = $db->prepare("SELECT `name`, `comment`, DATE_FORMAT(`date_created`, '%d/%m/%Y %H:%i') AS 'date_created' FROM `comments` WHERE issue_id = :id;");
+        $commentsQuery = $db->prepare("SELECT `name`, `comment`, `date_created` FROM `comments` WHERE issue_id = :id;");
         $commentsQuery->execute(['id' => $issue->id]);
         $commentsQuery->setFetchMode(PDO::FETCH_CLASS, Comment::class);
         $comments = $commentsQuery->fetchAll();
 
+        foreach($comments as $comment) {
+            $date = new \DateTime($comment->date_created);
+            $converted_date = $date->format("d/m/Y H:i");
+            $comment->date_created = $converted_date;
+        }
+
```

Comment:
You should keep the date stored in the entity as flexible as possible until the point that it is required to be displayed.

When is the date "displayed" in the comment? How do we control the way that it is displayed?